README.md

```diff
@@ -2,35 +2,327 @@
+
+While these problems could be somewhat softened by improving the existing notification system and making it easier for users to understand the progress of print jobs, allowing the applications to monitor them directly could also be beneficial for the users in a number of ways.
+* With more control over error handling, applications can wrap up raw printing errors into more elaborate descriptions (what went wrong and why?), suggest useful tips for troubleshooting common errors (potentially even going as far as providing links to vendor websites or user manuals for specific printers) or even prompt to print anew after the problem has been fixed.
+* Granting the application visibility into the progress of print jobs would allow the apps to present this information in a more graphical & convenient manner.
+
+## Main Motivating Use Case: Remote Printing
+The incremental improvements to the existing printing stack discussed in the previous sections do not address the crucial case of remote printing which has specific requirements that do not fit into the general web document-based model.
+
+* Remote Rendering
+
+  In many cases the document being printed is rendered entirely on the remote system; the implication is that the local system receives a finalized media (such as a PDF document) which must be printed as-is rather than a raw version of the document in HTML that can be tailored by the means of the browser to the final print media available locally using `@media print`. With the existing printing stack, the best-effort approach for a remote app would be to ask the user to provide a generic description of their printers' capabilities, but it would lead to a significantly worse user experience than letting the remote application do what it usually does with the appropriate knowledge of the capabilities of local printers -- this is where the proposed API steps in.
+
+* Customizing the Remote Print Job
+
+  The remote print job might come with additional parameters such as copies and duplex settings when configured by the user when printing via a virtual printer which should be honored by the local system with a user confirmation -- there's no way to forward this information to the local printing system other than prompt the user to select the necessary settings in the print dialog box. One option would be to ehnance `window.print()` by providing optional parameters to pre-populate values in the local print dialog as suggested earlier in the explainer; an alternative solution would be to declare a virtual printer that doesn't support any settings and force all of such decisions to be made through the local print dialog which might be a bit more concise. Either approach results in a tedious two-stage printing process involving way more user interactions that necessary.
```

Comment:
This is a case where it would be really helpful if we had mockups of the potential UI options to help illustrate the point. I think of this feature as similar to the "multiple downloads" permission. We absolutely can show the user a second print dialog to confirm the settings that they previously selected on the remote system. It's also likely useful to add some of these controls to `window.print()` or as enhancements to CSS `@media print` declarations. However, this API also offers a different flow where the user has made the decision up front to give the site broader discretion about how it uses the selected printer which could streamline the experience for trusted apps.

Reply:
Per the offline discussion will be addressed in a separate CL.